tests/unit/TestNetRemoteServiceClient.cxx

```diff
@@ -114,3 +115,74 @@ TEST_CASE("WifiAccessPointEnable API", "[basic][rpc][client][remote]")
         REQUIRE(result.status().has_details() == false);
     }
 }
+
+TEST_CASE("WifiAccessPointSetPhyType API", "[basic][rpc][client][remote]")
+{
+    using namespace Microsoft::Net::Remote;
+    using namespace Microsoft::Net::Remote::Service;
+    using namespace Microsoft::Net::Remote::Wifi;
+    using namespace Microsoft::Net::Wifi;
+    using namespace Microsoft::Net::Wifi::Test;
+
+    constexpr auto SsidName{ "TestWifiAccessPointSetPhyType" };
+    constexpr auto InterfaceName{ "TestWifiAccessPointSetPhyType" };
+
+    auto apManagerTest = std::make_shared<AccessPointManagerTest>();
+    Ieee80211AccessPointCapabilities apCapabilities{
+        .Protocols{ Ieee80211Protocol::B, Ieee80211Protocol::G, Ieee80211Protocol::N, Ieee80211Protocol::A, Ieee80211Protocol::AC, Ieee80211Protocol::AD, Ieee80211Protocol::AX, Ieee80211Protocol::AX }

```

Comment:
This has `AX` twice. Was `BE` intended, instead?

```suggestion
        .Protocols{ Ieee80211Protocol::B, Ieee80211Protocol::G, Ieee80211Protocol::N, Ieee80211Protocol::A, Ieee80211Protocol::AC, Ieee80211Protocol::AD, Ieee80211Protocol::AX, Ieee80211Protocol::BE }
```

Alternatively, if this is meant to be all of the protocols, we should define it as a constant (`constexpr`) at the top of the test file for all tests to use, and `magic_enum` can help with this:

```C++
namespace Microsoft::Net::Remote::Test
{
constexpr auto AllProtocols = magic_enum::enum_values<Microsoft::Net::Wifi::Ieee80211Protocol>();
} // Microsoft::Net::Remote::Test
```

Reply:
Just as a word of caution, `magic_enum::enum_values()` (and other magic_enum "magic") will only work with enums that have values <= 255. We can extend it to `UINT16_MAX - 1` if necessary (it will slow down the build, so we don't do that atm since it's not needed).

It's not a concern for this instance, but just want to give you a heads-up in case you decide to use such functionality later. Some examples where this doesn't work are the `Ieee80211AkmSuite` and `Ieee80211CipherSuite` enums since they have an underlying type and values of type `uint32_t`.